### PR TITLE
Update LDC, some submodules, improve checkout

### PR DESCRIPTION
**Pull request by Geod24**
So master have been broken for more than a month

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,10 +14,9 @@ matrix:
   allow_failures:
     - d: dmd-nightly
   include:
-    - d: ldc-1.11.0
+    - d: ldc
```
Comment (wilzbach) [inline .travis.yml]:
```suggestion
    - d: ldc-1.15.0
```

Let's lock this to avoid any sudden deployment errors (happened in the past).

Comment (Geod24) [reply .travis.yml]: Done

**Comment by Geod24** [comment]
Hum `mir-random` is still failing...

**Comment by Geod24** [comment]
@wilzbach : How bad would it be if we just went ahead and disabled the sanity test for the time being ?

**Comment by Geod24** [comment]
The Japanese submodule update was a shot in the dark in case some weird caching happened.
The English submodule update is because it was pointing to a commit outside of master and hence the instructions to locally compile led to a conflict.

**Comment by Geod24** [comment]
Waiting on https://github.com/dlang-tour/core-exec/pull/47 to see if it helps...

**Comment by Geod24** [comment]
@wilzbach : Despite my efforts, this is very hard to fix.
1) I can't make it work on OSX because the Docker image assume it's built on Linux
2) It uses LDC which has a lot of annoying defaults (e.g. forces you to install gcc and gold)
3) It spawns docker inside of docker, using an old release...

The only way I could see this being fixed "properly" involves a full rewrite of the `Dockerfile`, but will also lead to an increase in the image size.

I've already spent much more time than I should have on this, and my goal (which was to have Korean language deployed on the official website) is nowhere near being achieved.
I removing the controversial commits, and I hope you'll merge this PR, even if it does not fix everything.

I'd like to remind you that I effectively wasted many days trying to get this to work, just for a sanity check which is not even mandatory for the tour to work.

**Comment by Geod24** [comment]
@PetarKirov : Can you review this ?

**Comment by PetarKirov** [comment]
Finally a green build! Unbelievable :smile:

**Pull request merged**